src/custom-elements/docs/docs.json

```diff
@@ -2088,6 +2088,28 @@
           "optional": false,
           "required": false
         },
+        {
+          "name": "mode",
```

Comment:
@janibolkvadze @jllr Thought on using `variant` instead of mode? We use variant on the button component.